src/linux/system.rs

```diff
@@ -154,9 +154,11 @@ pub struct System {
     process_list: Process,
     mem_total: u64,
     mem_free: u64,
+    mem_available_found: bool,
```

Comment:
Is there any advantage at storing it? Why not just create a boolean in the function directly?